docker-compose.prod.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.9"
+
+services:
+  fossolgyui_server:

```

Comment:
How do you feel about adding FOSSology server as a service as well? Will it be helpful in your opinion for anyone setting up their environment for first time?

Reply:
It can use the image `fossology/fossology:latest` and do following mounts (as per https://github.com/fossology/fossology/blob/master/docker-compose.yml):
```yml
volumes:
  - repository:/srv/fossology/repository/
  - database:/var/lib/postgresql/data
```

Reply:
Since, FOSSology server serves as only an API backend, I don't think there would be any harm in adding this in. This would actually resolve a lot of the issues that pop-up on Slack regarding set up of FOSSology server considering it's in a different repo entirely. I'll add this in.